pkg/occlient/occlient.go

```diff
@@ -1184,16 +1221,45 @@ func (c *Client) GetBuildConfigFromName(name string, project string) (*buildv1.B
 	return bc, nil
 }
 
-// GetClusterServiceClasses queries the service service catalog to get all the
-// currently available cluster service classes
+// GetClusterServiceClasses queries the service service catalog to get available clusterServiceClasses
 func (c *Client) GetClusterServiceClasses() ([]scv1beta1.ClusterServiceClass, error) {
-	classList, err := c.serviceCatalogClient.ClusterServiceClasses().List(metav1.ListOptions{})
+	// TODO: Remove `FieldSelector` from ListOptions when we are confident with Ansible Service Broker
+	classList, err := c.serviceCatalogClient.ClusterServiceClasses().List(metav1.ListOptions{FieldSelector: "spec.clusterServiceBrokerName=template-service-broker"})
```

Comment:
Why do you hard code this value as different brokers could be installed on Openshift and not only Openshift Template Broker but ASB, ... ?

Reply:
Initially, when I tried with ASB, it was behaving weirdly, so I thought of going with TSB as of now

Reply:
even most of the APBs have `clusterserviceplan`, initially I would like to experiment with TSB and then jump into ASB